src/test/java/com/connexta/store/StoreITests.java

```diff
@@ -181,7 +178,8 @@ void beforeEach() throws Exception {
     mockFileServer = new MockWebServer();
     mockFileServer.start();
 
-    indexMockRestServiceServer = MockRestServiceServer.createServer(nonBufferingRestTemplate);
+    mockIndexServer = new MockWebServer();
+    mockIndexServer.start();
```

Comment:
❓ Why is `mockIndexServer` created? I don't see it being used anywhere?

Reply:
Whoops, I should be returning a successful request from here. Not sure why the test worked! I wonder if default is success.